src/bin/miri.rs

```diff
@@ -487,6 +487,13 @@ fn main() {
                 "full" => BacktraceStyle::Full,
                 _ => panic!("-Zmiri-backtrace may only be 0, 1, or full"),
             };
+        } else if let Some(param) = arg.strip_prefix("-Zmiri-external_c_so_file=") {
+            let filename = param.to_string();
+            if std::path::Path::new(&filename).exists() {
+                miri_config.external_c_so_file = Some(filename);
+            } else {
+                panic!("-Zmiri-external_c_so_file requires a valid path to a file");
```

Comment:
Maybe "path {} does not exist"? The error right now sounds like an argument wasn't provided, which isn't the issue.

Reply:
good point